develop.md

```diff
@@ -31,4 +31,8 @@ For example, in order to debug the test [java-compiler-executable](src/it/java-c
 *Note* that you have to run `mvn invoker:install` to debug the latest changes in your code!
 
 #### Integration Tests
-The Integration tests are located in its, and they can be run as a separate Maven project: `cd its && mvn verify`
+The Integration tests are located in `its`.
+
+Before running them, you have to package the `property-dump-plugin` by running the following command from the root of the project: `mvn -f property-dump-plugin/pom.xml clean package`. 
+
```

Comment:
There is no need for that much spacing but that already makes the text easier to mana
```suggestion
Before running them, you have to package the `property-dump-plugin` by running the following command from the root of the project: `mvn -f property-dump-plugin/pom.xml clean package`. 
```